blockchain/config.go

```diff
@@ -32,6 +32,9 @@ type (
 		BloomfilterIndexDBPath string           `yaml:"bloomfilterIndexDBPath"`
 		CandidateIndexDBPath   string           `yaml:"candidateIndexDBPath"`
 		StakingIndexDBPath     string           `yaml:"stakingIndexDBPath"`
+		SGDIndexDBPath         string           `yaml:"sgdIndexDBPath"`
+		SGDIndexCacheSize      int              `yaml:"sgdIndexCacheSize"`
```

Comment:
this needs not be in the yaml, can be a constant in the `New()`

Reply:
https://github.com/iotexproject/iotex-core/pull/3845/files#diff-c65dd5a657c73eb89458c69a82a25fc0993655650b3ad477136955c31e50e2bbR279-R283

you mean hardcoding here? `cacheSize` => `1000`
```go

		kvStore, err := db.CreateKVStoreWithCache(builder.cfg.DB, builder.cfg.Chain.SGDIndexDBPath, builder.cfg.Chain.SGDIndexCacheSize)
		if err != nil {
			return err
		}
		builder.cs.sgdIndexer = blockindex.NewSGDRegistry(builder.cfg.Chain.SGDContract, kvStore)
```